index.js

```diff
@@ -119,13 +119,11 @@ module.exports = {
 
       since                    : since,
 
-      getPublicKey             : ssb.getPublicKey,
       latest                   : ssb.latest,
       getLatest                : valid.async(ssb.getLatest, 'feedId'),
       latestSequence           : valid.async(ssb.latestSequence, 'feedId'),
       createFeed               : ssb.createFeed,
       whoami                   : function () { return { id: feed.id } },
-      query                    : ssb.query,
```

Comment:
Are these not used anywhere?

Reply:
query isn't even set! latest,getLatest,latestSequence are used in some tests. createFeed is mostly used in tests. this is the worst case because changing the tests isn't directly useful, but it would allow us to remove a bunch of code.